clients/drcachesim/tests/scheduler_launcher.cpp

```diff
@@ -206,6 +214,12 @@ simulate_core(int ordinal, scheduler_t::stream_t *stream, const scheduler_t &sch
                 std::cerr << line.str();
             }
         }
+        if (type_is_instr(record.instr.type))
+            ++cur_segment_instrs;
+        if (cur_segment_instrs == op_print_every.get_value()) {
+            thread_sequence += 'A' + static_cast<char>(input % 26);
+            cur_segment_instrs = 0;
+        }
         prev_input = input;
```

Comment:
nit: move this inside the `input != prev_input` if-block.

Reply:
Done.